config/app-config.cfg.example

```diff
@@ -39,6 +39,8 @@ enabled = false
 name =
 email =
 host =
+port =
+usessl =
```

Comment:
I would name it just `ssl` for simplification

Reply:
Solved in https://github.com/ds-wizard/dsw-server/pull/42/commits/3b85b9807c60716a599351ead0ff5d672863e81e